modules/router/src/test/services/config.spec.ts

```diff
@@ -23,4 +45,70 @@ describe("config.ts", () => {
       expect(res.getValue()).to.be.deep.eq(config.rebalanceProfiles[0]);
     });
   });
+
+  describe("getMatchingSwap", () => {});
+
+  describe("getSwapFees", () => {});
+
+  describe("getMappedAssets", () => {
+    it("happy: getMappedAssets", () => {
+      const res = getMappedAssets(fromAssetId, fromChainId);
+      console.log(res);
+    });
+  });
+
+  describe("getPriceImpact", () => {
+    it("happy: getMappedAssets 1:1", () => {
+      const marketPrice = parseEther("1");
+      const estimatedPrice = parseEther("1");
+      const res = getPriceImpact(marketPrice, estimatedPrice);
+      console.log(res);
+    });
+
+    it("happy: getMappedAssets 1:1.2", () => {
+      const marketPrice = parseEther("1");
+      const estimatedPrice = parseEther("1.2");
+      const res = getPriceImpact(marketPrice, estimatedPrice);
+      console.log(res.toString());
```

Comment:
I think we should actually ensure desired behavior here for above three methods, no? If this is just a TODO item, I'll take care of it